src/platform/datapath_raw_xdp_win.c

```diff
@@ -371,6 +382,38 @@ CxPlatDpRawInterfaceUninitialize(
     #pragma warning(pop)
 }
 
+QUIC_STATUS
+GetTxOffloadConfig(
+    _In_ CXPLAT_QUEUE* Queue,
+    _Out_ uint32_t* TxChecksumOffload
+    )
+{
+    QUIC_STATUS Status;
+    XDP_CHECKSUM_CONFIGURATION ChecksumConfig;
+    uint32_t OptionLength = sizeof(ChecksumConfig);
+
+    Status =
+        XskGetSockopt(
+            Queue->TxXsk, XSK_SOCKOPT_TX_OFFLOAD_CURRENT_CONFIG_CHECKSUM, &ChecksumConfig,
+            &OptionLength);
+    if (QUIC_FAILED(Status) ||
```

Comment:
We should log at the lowest point (not at the caller). Additionally, I think we should have separate logs to distinguish between failure was returned vs unsupported version.